Resources/keywords.robot

```diff
@@ -6,12 +9,11 @@ Begin Web Test
     Call Method    ${chrome_options}    add_argument    --headless
     Call Method    ${chrome_options}    add_argument    --disable-gpu
     Call Method    ${chrome_options}    add_argument    --no-sandbox
-    #Call Method    ${chrome_options}    add_argument    --start-maximized
+    Call Method    ${chrome_options}    add_argument    --start-maximized
     Call Method    ${chrome_options}    add_argument    ${WINDOW}
-    #Call Method    ${chrome_options}    add_argument    window-size=1920,1080
     Create Webdriver    Chrome    chrome_options=${chrome_options}
-    #Set Window Size  ${1920}  ${1080}
-    #Set Window Size  1920  1080
+    Maximize Browser Window
```

Comment:
This is redundant due to option on row 12

Reply:
Ok